python_modules/libraries/dagster-fivetran/dagster_fivetran/resources.py

```diff
@@ -1007,14 +1007,17 @@ def _generate_materialization(
                             schema=schema.name_in_destination,
                             table=table.name_in_destination,
                         ),
-                        "schema_source_name": schema_source_name,
-                        "table_source_name": table_source_name,
+                        **FivetranMetadataSet(
+                            connector_id=connector.id,
+                            destination_schema_name=schema.name_in_destination,
```

Comment:
The legacy code used the source schema and source table when calling the [Table Column Configs endpoint](https://fivetran.com/docs/rest-api/api-reference/connector-schema/connector-column-config), which is incorrect according to the current documentation:

> schema   string required
> The database schema name within your destination

Instead we are now using the destination schema and table, and we are using the FivetranMetadataSet to add the metadata.